test/integration/short/prepare-simulator-tests.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+var assert = require('assert');
+var util = require('util');
+
+var Client = require('../../../lib/client');
+var types = require('../../../lib/types/index');
+var utils = require('../../../lib/utils');
+var helper = require('../../test-helper');
+var reconnection = require('../../../lib/policies/reconnection');
+var simulacron = require('../simulacron');
+
+describe('Client #prepare #simulacron', function () {
```

Comment:
The `#name()` notation is used to represent an instance method. I think using like `'Client'` would be fine.